swagger.yaml

```diff
@@ -4476,6 +4480,7 @@ paths:
         - userAuth: [admin, read-only-admin, user]
         - agentAuth: []
         - urlAuth: []
+        - imageAuth: []
```

Comment:
what will happen before we have the image service
What will happen with existing infra-envs?

Reply:
> what will happen before we have the image service

This API will still accept imageAuth tokens if they're present, but we won't issue any before the image service is deployed.

> What will happen with existing infra-envs?

Need to create a migration to create keys for them. Put a hold on the PR until I get to it.